src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -26947,6 +26947,40 @@ public static bool TakeOutParam<T>(T y, out T x)
             Assert.Null(model.GetAliasInfo(x1Decl.Type()));
         }
 
+        [Fact, WorkItem(14717, "https://github.com/dotnet/roslyn/issues/14717")]
+        public void ExpressionVariableInCase()
```

Comment:
I think we should do the extra testing now.  There have been a number of late breaking error reports in this area.  Let's lock down as much as possible for RC.